### PR TITLE
:book: Fix documentation typos

### PR DESCRIPTION
**Pull request by omahs**
Fix typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ metrics. Prominent projects that use Scorecard include:
 
 ### View a Project's Score
 
-To see scores for projects regually scanned by Scorecard, navigate to the webviewer, replacing the placeholder text with the platform, user/org, and repository name: 
+To see scores for projects reguarly scanned by Scorecard, navigate to the webviewer, replacing the placeholder text with the platform, user/org, and repository name:
```
Comment (gossts) [inline README.md:97]:
```suggestion
To see scores for projects regularly scanned by Scorecard, navigate to the webviewer, replacing the placeholder text with the platform, user/org, and repository name: 
```

Comment (raghavkaul) [inline README.md:97]:
```suggestion
To see scores for projects regularly scanned by Scorecard, navigate to the webviewer, replacing the placeholder text with the platform, user/org, and repository name: 
```

File: docs/checks.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Different types of branch protection protect against different risks:
 
     - requiring two or more reviewers protects even more from the insider risk 
     whereby a compromised contributor can be used by an attacker to LGTM 
-    the attacker PR and inject a malicious code as if it was legitm.
```
Comment (spencerschrock) [inline docs/checks.md:86]: Please see our [CONTRIBUTING.md section](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#updating-docs) on modifying this file.

Comment (omahs) [reply docs/checks.md:86]: Hey @spencerschrock, I added the change to the corresponding `checks.yaml` file

**Pull request merged**